### DISCUSSION TITLE
`className` ts error on motion components

### DISCUSSION THREAD

**Discussion by dragidavid** [motiondivision/motion#2850] [Q&A]
Hi, I just recently bumped nextjs and react to 15 and 19 respectively and even though the app starts and seems to work fine, I'm getting this weird ts error for motion components, for the `className` prop out of everything 🤔 

```
Type '{ children: Element; key: string; initial: { opacity: number; filter: string; }; animate: { opacity: number; filter: string; }; transition: { opacity: { duration: number; delay: number; }; filter: { ...; }; }; className: string; style: CSSProperties; "aria-hidden": true; }' is not assignable to type 'IntrinsicAttributes & HTMLAttributesWithoutMotionProps<unknown, unknown> & MotionProps & RefAttributes<unknown>'.
  Property 'className' does not exist on type 'IntrinsicAttributes & HTMLAttributesWithoutMotionProps<unknown, unknown> & MotionProps & RefAttributes<unknown>'.ts(2322)
```

Anyone had something similar happening recently? I'm assuming its a new version issue but haven't seen anyone else reporting it.

![Screenshot 2024-10-27 at 21 45 46](https://github.com/user-attachments/assets/bfdf4d09-c542-4b52-9410-91e968e41840)

**Comment by ScreamZ** [comment]
The same issue, seems related to react 19 ? @mattgperry

**Comment by hades200082** [comment]
It looks like this is the same issue as #2843

**Reply by ScreamZ** [reply]
Fuck nextjs releasing stable versions based on rc/alpha 😑

**Reply by hades200082** [reply]
tbh, if they didn't we'd all be beholden to Facebook actually releasing the next version of React which seems to be taking a while

**Reply by ScreamZ** [reply]
Sure, but Facebook wait for things to get stable, this is one of the most used library in the world, they can't released things unstable 😅
